modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -58,6 +58,7 @@ def config
   def schemas
     a = []
     a << generic_schemas('#/components/schemas')
+    a << contestable_issues_schema('#/components/schemas')
```

Comment:
ooc, why does this move to the top?

Reply:
At the bottom of this method we merge duplicate keys & then sort the keys alphabetically:

```ruby
 a.reduce(&:merge).sort_by { |k, _| k.to_s.downcase }.to_h
```